ftest.sh

```diff
@@ -145,21 +145,26 @@ fi
 trap 'set +e; cleanup' EXIT
 
 CLUSH_ARGS=($CLUSH_ARGS)
-
-DAOS_BASE=${SL_PREFIX%/install}
 if ! clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh -S \
     -w "$(IFS=','; echo "${nodes[*]}")" "set -ex
-# allow core files to be generated
+# allow ulimit files
 sudo bash -c \"set -ex
-if [ \\\"\\\$(ulimit -c)\\\" != \\\"unlimited\\\" ]; then
-    echo \\\"*  soft  core  unlimited\\\" >> /etc/security/limits.conf
-fi
 if [ \\\"\\\$(ulimit -l)\\\" != \\\"unlimited\\\" ]; then
     echo \\\"*  soft  memlock  unlimited\\\" >> /etc/security/limits.conf
     echo \\\"root  soft  memlock  unlimited\\\" >> /etc/security/limits.conf
     echo \\\"session required pam_limits.so\\\" > /etc/pam.d/common-session-noninteractive
     echo \\\"session required pam_limits.so\\\" > /etc/pam.d/common-session
-    echo \\\"session required pam_limits.so\\\" >> /etc/pam.d/login
+    echo \\\"session required pam_limits.so\\\" >> /etc/pam.d/login\"
+fi"
+fi
+
```

Comment:
Having to separate ssh sessions to do the same configuration is not going to change anything.

Reply:
Any recommendation on how to handle it?  Based on this article [https://superuser.com/questions/740000/modify-and-apply-limits-conf-without-reboot], I thought I have to set the memlock value, update /etc/pam.d/common-session and log back in..

Reply:
The (shell) _session_  that the updating of `/etc/security/limits.conf` files is done in is not the same session that the tests are launched in so the goal of that link you pasted is already being achieved in the existing structure.

Additionally, `/etc/pam.d/common-session` is not used/valid on EL7, and EL7 already includes `pam_limits.so` in it's PAM stack.

Again, I would encourage you to experiment with all of this on a couple of local nodes, as it will be much easier to investigate and understand the effects of the commands you are using than continuing to throw things at CI where investigating the results of your changes is pretty difficult.